### PR TITLE
[RUM-4910] Allow disable of RUM Action accessibility label name resolving

### PR DESCRIPTION
**Pull request by marco-saia-datadog**
### What does this PR do?

When available, auto-instrumented actions get their name from views `accessibilityLabel` prop.

With this PR we want to provide an option to disable this behaviour, in case accessibility labels contain any sensible data that should not be collected.

### Notes

- I am considering renaming it to `useAccessibilityLabels` (plural) or finding a better name.

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)
- [ ] If this PR is auto-generated, please make sure also to manually update the code related to the change

### CODE REVIEW

**Comment by louiszawadzki** [comment]
Hello @marco-saia-datadog :) 

Do you have any update on this PR?

File: packages/core/android/src/test/resources/input/complete-configuration.json [hunk 1]
```diff
@@ -45,4 +49,4 @@
         "version": "2.3.1",
         "vitalsUpdateFrequency": "NEVER"
     }
-}
+}
```
Comment (0xnm) [inline packages/core/android/src/test/resources/input/complete-configuration.json]:
```suggestion
}

```

File: packages/core/datadog-configuration.schema.json [hunk 2]
```diff
@@ -194,7 +235,11 @@
...
-            "required": ["clientToken", "env", "applicationId"]
+            "required": [
+                "clientToken",
+                "env",
+                "applicationId"
+            ]
         }
     }
-}
+}
```
Comment (0xnm) [inline packages/core/datadog-configuration.schema.json]: should we add a formatter for JSON in this project? seems nothing adds newline.

```suggestion
}

```

File: packages/core/ios/Tests/Fixtures/complete-configuration.json [hunk 3]
```diff
@@ -45,4 +49,4 @@
         "version": "2.3.1",
         "vitalsUpdateFrequency": "NEVER"
     }
-}
+}
```
Comment (0xnm) [inline packages/core/ios/Tests/Fixtures/complete-configuration.json]:
```suggestion
}

```

**Review by 0xnm — APPROVED** [review]

**Review by ganeshnj — APPROVED** [review]

**Comment by marco-saia-datadog** [comment]
Hello @louiszawadzki 😃 👋 

We have released this in version [v2.4.0](https://github.com/DataDog/dd-sdk-reactnative/releases/tag/2.4.0).

**Comment by louiszawadzki** [comment]
Thanks a lot @marco-saia-datadog and all the team @0xnm @ganeshnj !! 🙇

**Pull request merged**